.github/workflows/build-and-deploy.yml

```diff
@@ -1,5 +1,8 @@
 name: Build and Deploy
 on:
+on:
```

Comment:
I am not an expert in github action, but there are two times `on:` ?

Reply:
Yes 😅 

This proves that even if you are not an expert in a field you can always provide value by reviewing the contribution 👍